libs/nest/audit/src/lib/audit.service.ts

```diff
@@ -1,24 +1,24 @@
 import winston from 'winston'
 import WinstonCloudWatch from 'winston-cloudwatch'
-import { format, TransformableInfo } from 'logform'
+import { TransformableInfo } from 'logform'
 import { createHash } from 'crypto'
 import { Inject, Injectable } from '@nestjs/common'
 import type { Logger } from '@island.is/logging'
 import { LOGGER_PROVIDER } from '@island.is/logging'
-import type { User } from '@island.is/auth-nest-tools'
+import type { Auth } from '@island.is/auth-nest-tools'
 import type { AuditOptions } from './audit.options'
 import { AUDIT_OPTIONS } from './audit.options'
 
 export interface AuditMessage {
-  user: User
+  user: Auth
```

Comment:
Ideally all of these are changed to be `auth: Auth` rather than `user: Auth`. But I guess that's a pretty big change across the monorepo. 

Maybe we can support both for backwards compatibility and deprecate `user`? Or just do a big PR :)

Reply:
I actually started doing that, but realised it would be a pretty big PR so a backed away from it. But I'm up for a big PR so just let's do it!